src/components/login/LoggedInModal.tsx

```diff
@@ -0,0 +1,50 @@
+import { Modal, Zoom } from "@mui/material";
+import { Box } from "@mui/system";
+import React, { useContext } from "react";
+import CheckCircleIcon from "@mui/icons-material/CheckCircle";
+import classes from "../../pages/login/index.module.css";
+import Link from "next/link";
+import { CustomButton } from "../CustomMUI/CustomUI";
+import { LoggedInContext } from "./IsLoggedIn";
+import { modalProps } from "../../types";
```

Comment:
This type is only used in this module, and is designed specifically for the `LoggedInModal` component.

I would define the type here instead of importing it from some other module, and call it something that clearly relates it to the component, e.g. `LoggedInModalProps`.